### PR TITLE
Vagrant deployment

### PR DESCRIPTION
**Pull request by kotfic**
Currently passes smoke tests for vagrant deployment without breaking ec2 deployment,  including downloading and building paraview, moab etc

### CODE REVIEW

File: ansible/roles/cumulus/tasks/main.yml [hunk 1]
```diff
@@ -41,10 +45,12 @@
   sudo: yes
   tags: cumulus
 
+- name: Foo
+  debug: msg="{{ get_private_ip }}"
+
```
Comment (cjh1) [inline ansible/roles/cumulus/tasks/main.yml]: I am guessing this can be removed?

**Comment by cjh1** [comment]
Looks good can we add a little documentation to the readme, documenting the command to run and parameters required?

File: ansible/roles/cumulus/tasks/main.yml (hunk 1 above)
Comment (kotfic) [inline ansible/roles/cumulus/tasks/main.yml]: Whoops!

**Comment by jourdain** [comment]
We don't have any need of Moab anymore. You should be able to remove that dependency. 

Sorry

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  end
+
+  config.vm.provision "ansible" do |ansible|
+    ansible.groups = {
+      "all" => ["cumulus"],
+      "cumulus" => ["cumulus"],
+      "girder" => ["cumulus"],
+      "hpccloud" => ["cumulus"],
+      "paraview" => ["cumulus"]
+    }
```
Comment (cjh1) [inline Vagrantfile:44]: If I want to deploy the pvwlauncher role, would I add it to this list?

**Comment by cjh1** [comment]
LGTM, will merge and improvements can be done on follow on PRs.

**Pull request merged**